csharp/BUILDING.md

```diff
@@ -126,7 +126,7 @@ necessary.
 
 ### Linux and macOS Build Requirements
 
-You need the [.NET 6.0 SDK][4] or [.NET 7.0 SDK][5] to build Ice for .NET from source.
+You need the [.NET 8.0 SDK][4] or [.NET 7.0 SDK][5] to build Ice for .NET from source.
```

Comment:
I thought we only wanted to support the 8 and 10 SDKs. We'll do this in a follow-up?